util/tools/translator.ts

```diff
@@ -11,6 +11,17 @@ import ora, { Ora } from "ora";
  * @author Bas950 <me@bas950.com>
  */
 
+/**
+ * USING THE TRANSLATOR TOOL
+ *
+ * - Open up a terminal and run yarn translator
+ *	- When prompted, select the Language Code for the language you are translated (e.g `ga_IE` for Irish)
+ *	- Once selected. The script will first cache all presences metadata files, it will then go through each one and remove the files containing the language you inputted from the cache.
```

Comment:
This period makes no sense here, also no one cares about the technical details since most people who use this aren't gonna know how to code, so this line can be removed and the following one can be adjusted slightly to make the transition clearer

Reply:
> Add the translator guide to the translator file as there isn't a proper place for it. Should be moved to PreMiD/pmd soon though
I won't make changes to this for this reason